package.json

```diff
@@ -20,7 +20,7 @@
 		"fix": "pnpm run prettify && dotenv -- turbo check:lint -- --fix",
 		"prettify": "prettier . --write --ignore-unknown",
 		"test": "vitest run --no-file-parallelism && dotenv -- turbo test --filter=wrangler --filter=miniflare --filter=kv-asset-handler --filter=@cloudflare/vitest-pool-workers --filter=@cloudflare/vitest-pool-workers-examples",
-		"test:ci": "dotenv -- turbo test:ci",
+		"test:ci": "dotenv -- turbo test:ci --concurrency 100%",
```

Comment:
Assuming the available cores is > 2, I think 50% would be better, considering on avg at least 2 processes are started per test